src/serviceWorker/arkade-lightning-message-handler.ts

```
@@ -0,0 +1,783 @@
+            ...res,
+            tag: this.messageTag,
+        } as ArkadeLightningUpdaterResponse;
+    }
+
+    private async broadcastEvent(
+        event: SwapManagerEventMessage
+    ): Promise<void> {
+        const sw: any = self as any;
+        if (!sw?.clients?.matchAll) return;
+        const clients = await sw.clients.matchAll();
+        clients.forEach((client: any) => client.postMessage(event));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)